scripts/check_manifest.py

```
@@ -0,0 +1,149 @@
+    root = arguments.root.resolve()
+    registry = read_registry(root)
+    selected, sections = read_manifest(arguments.manifest)
+
+    problems = []
+
+    for token in selected:
+        if token not in sections:
+            problems.append(
+                f"{token}: named in 'Selected platforms' but has no section — "
+                f"the whole platform is missing from the manifest"
+            )
```

Ruff (0.16.0)

[warning] Use a list comprehension to create a transformed list

(PERF401)